common.mk

```diff
@@ -20,7 +20,7 @@ ifeq ($(UNAME_S),Darwin)
     GMP_DIR?=/opt/homebrew/opt/gmp
   endif
   LIB_SUF=dylib
-  OPENSSL_DIR?=/usr/local/opt/openssl
+  OPENSSL_DIR?=/opt/homebrew/opt/openssl@1.1
```

Comment:
Consider setting this depending upon the OS here
https://github.com/harmony-one/mcl/blob/83e553ec4ed1c231fcfeaa9d54cbef0f44ef8637/common.mk#L18-L20